imports/plugins/included/payments-stripe/client/checkout/containers/stripePaymentFormContainer.js

```diff
@@ -0,0 +1,53 @@
+import { Meteor } from "meteor/meteor";
```

Comment:
Import order

Reply:
Fixed and will keep this in mind moving forward!